### DISCUSSION TITLE
Installing Python packages in QGIS 3 (for Windows)

### DISCUSSION THREAD

**Discussion by sjib** [QGEP/QGEP#708] [Q&A]
When you need to install additional python libraries and the data model tool can not handle it automatically then follow these instructions:
https://landscapearchaeology.org/2018/installing-python-packages-in-qgis-3-for-windows/

**Comment by sjib** [comment]
@TriformMoritz Thanks for the hint. Any suggestion where to integrate this info in the documentation? 
@urskaufmann How far are you with the documentation of the data model tool? Could this be placed there?

**Reply by urskaufmann** [reply]
I asked @olivierdalang to write the docu for the data model tool, because I know only the user side and do know only my configuration. Nothing heard since then (september)

**Reply by olivierdalang** [reply]
This is now done : https://github.com/QGEP/docs/pull/172
